File: src/Distinct.java
import java.util.HashSet;

public class Distinct {
    public int solution(int[] A){

        HashSet<Integer> hash = new HashSet<>();
        for(int a : A){
            hash.add(a);

        }
        return hash.size();

    }
    public static void main(String[] args){
        Distinct D = new Distinct();
        System.out.println(D.solution(new int[]{2,1,1,2,3,1}));
    }

}

File: src/JS/Distinct.js
function Distinct(A) {
  A.sort();
  let set = new Set();
  for (let i = 0; i < A.length; i++) {
    set.add(A[i]);
  }
  return set.size;
}

console.log(Distinct([2, 1, 1, 2, 3, 1]));
